### PR TITLE
meta : add encode benchmark

### PR DESCRIPTION
**Pull request by orenyodfat**

### CODE REVIEW

File: meta_test.go [hunk 1]
```diff
@@ -94,3 +95,54 @@ func TestEncodeDecode(t *testing.T) {
...
 	}
 }
+
+func benchmarkEncode(n int, t *testing.B) {
+	type test struct {
+		Array []string `json:"array"`
+	}
+	v := &test{}
+	for j := 0; j < n; j++ {
+		v.Array = append(v.Array, "t")
```
Comment (lmars) [inline meta_test.go]: You could just make this a `n` length string using [`strings.Repeat`](https://golang.org/pkg/strings/#Repeat).

File: meta_test.go [hunk 2]
```diff
@@ -94,3 +95,54 @@ func TestEncodeDecode(t *testing.T) {
...
+	}
+	t.ReportAllocs()
+	for i := 0; i < t.N; i++ {
```
Comment (lmars) [inline meta_test.go]: Reset the timer with [`B.ResetTimer`](https://golang.org/pkg/testing/#B.ResetTimer).

Comment (orenyodfat) [reply meta_test.go]: Add ResetTimer at 51c00f3 though it does not have any effect on the result.

**Comment by orenyodfat** [comment]
Thanks @lmars ,  code is updated.

**Pull request merged**